package.json

```diff
@@ -60,7 +61,8 @@
     "lint:quiet": "npm run lint -- --quiet",
     "eject": "react-scripts eject",
     "storybook": "start-storybook -p 9009 -s public",
-    "build-storybook": "build-storybook -s public"
+    "build-storybook": "build-storybook -s public",
+    "swagger:generate-schema": "dtsgen -o ./src/__generated__/apiSchema.d.ts -n API --url https://acc.api.top.amsterdam.nl/api/v1/swagger/?format=openapi"
```

Comment:
We probably need:
`"swagger:generate-schema:acc": "dtsgen -o ./src/__generated__/apiSchema.d.ts -n API --url https://acc.api.top.amsterdam.nl/api/v1/swagger/?format=openapi"`
`"swagger:generate-schema": "dtsgen -o ./src/__generated__/apiSchema.d.ts -n API --url https://api.top.amsterdam.nl/api/v1/swagger/?format=openapi"`

Reply:
Good one, I'll add the other one as well.